arithmetization/src/main/java/net/consensys/linea/zktracer/module/blake2fmodexpdata/Blake2fModexpDataOperation.java

```diff
@@ -200,7 +200,7 @@ private UnsignedByte[] getHubStampDiffBytes(Bytes currentHubStamp) {
     BigInteger hubStampDiff = hubStampBigInt.subtract(prevHubStampBigInt).subtract(BigInteger.ONE);
 
     Preconditions.checkArgument(
-        hubStampDiff.compareTo(BigInteger.valueOf(256 ^ 6)) < 0,
+        hubStampDiff.compareTo(BigInteger.valueOf((long) Math.pow(256, 6))) < 0,
```

Comment:
not sure it's usefull as it will break anyway if the hubStamp is greater than 256**6

Reply:
Why would it break if the HUB stamp is larger than that ? Isn't it an int ?

Reply:
HUB STAMP / CONTEXT_NUMBER columns are typed as an `:i48`